app/controllers/gi_controller.rb

```diff
@@ -11,6 +11,8 @@ def client
   end
 
   def scrubbed_params
-    params.except(:action, :controller, :format)
+    params.except(:action, :controller, :format).transform_values do |v|
+      v.encode!('UTF-8', invalid: :replace, undef: :replace, replace: '')
```

Comment:
I would think we need to get deeper than transform_values allows, what happens if value is a hash with additional nested objects? 

Additionally, would a request spec be useful here?

I'm curious also what the underlying issue is. Do you think it's users passing in attributes that have some other type of encoding? I was under the impression that rails enforced utf-8 by default since Rails 4+, but I might be wrong. 

The biggest issue I see overall with transforming the data in this manner is that we replace invalid data with a blank. Changing a users query parameters is less than desirable, instead we should maybe raise an error or exception?